spaces/S000107/properties/P000041.md

```diff
@@ -0,0 +1,10 @@
+---
+space: S000107
+property: P000041
+value: false
+refs:
+- mathse: 5012784
+  name: Answer to "Is $\ell^\infty$ with box topology connected?"
+---
+
+By {{mathse:5012784}} the component of $x$ is $A = \{y : y_n = x_n\text{ for all but finitely many }n\}$. Since $\text{int}(A) = \emptyset$, it follows that $x$ has no connected neighbourhoods.
```

Comment:
```suggestion
By {{mathse:5012784}} the connected component of an arbitrary point $x\in X$ is $A = \{y : y_n = x_n\text{ for all but finitely many }n\}$. Since $\text{int}(A) = \emptyset$, it follows that $x$ has no connected neighbourhoods.
```